exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/partitionsender/PartitionerDecorator.java

```diff
@@ -26,13 +26,13 @@
 import org.apache.drill.exec.ops.FragmentContext;
 import org.apache.drill.exec.ops.OperatorStats;
 import org.apache.drill.exec.record.RecordBatch;
-
```

Comment:
I might have hit the organize imports on Eclipse (because of unused imports), which follows Sun conventions regarding import order. I can revert the change and do an import cleanup in a different PR.